### PR TITLE
[ADD] project_timeline

### PR DESCRIPTION
**Pull request by pedrobaeza**
# Timeline view for projects

This module extends the standard tasks actions to add the timeline view to
them, and also adds the needed fields in the view for handling them.
# Usage
- Go to _Project > Project > Tasks_.
- Click on the icon with a clock.
- You will see the tasks in the new view.

### CODE REVIEW

File: oca_dependencies.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # List the OCA project dependencies, one per line
 # Add a repository url and branch if you need a forked version
 account-analytic
+web https://github.com/Tecnativa/web 9.0-web_timeline
```
Comment (Yajo) [inline oca_dependencies.txt]: I''m afraid runbot is ignoring this.

**Comment by pedrobaeza** [comment]
This one is ready to be reviewed, as the dependency has been merged. Basically, it has been the module for testing the view, so it should be OK. I have added demo data for the fields involved to have a better timeline view.

@dreispt @lmignon @chienandalu @luismontalba Can you please review?

**Comment by pedrobaeza** [comment]
We should deactivate codacy, as it doesn't perform well on Odoo modules...

**Review by luismontalba — APPROVED** [review]
Functionality tested on runbot. Nice view, thanks!

**Review by dreispt — APPROVED** [review]

**Pull request merged**